index.html

```diff
@@ -230,52 +230,79 @@ <h4 class="work-item-title">Multi-Post Stories</h4>
       </section>
       <section class="contact">
         <p class="contact-title desktop">Let's chat!</p>
-        <p class="contact-title mobile">CONTACT US</p>
+        <p class="contact-title mobile">CONTACT ME</p>
         <p class="contact-description">
           If you have an application you are interested in developing, a feature
           that you need built or a project that needs coding. I’d love to help
           with it
         </p>
-        <form class="contact-form">
+        <form
+          class="contact-form"
+          action="https://formspree.io/f/xvolvllr"
+          method="POST"
+        >
           <input
             type="text"
             id="fullname"
             name="fullname"
             placeholder="Enter Full Name"
+            maxlength="30"
```

Comment:
- [ ] Please use correct paddings for the input fields and make sure it matches with the Figma design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/166552115-9618bfe8-b86c-4bcb-aea6-24753fdb4e28.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/166552077-0e8557b9-078d-40ba-9bca-bb3098df5554.png)